packages/genesys-spark-components/src/components/beta/gux-tooltip-beta/tests/gux-tooltip.e2e.ts

```diff
@@ -29,6 +29,8 @@ describe('gux-tooltip-beta', () => {
           'gux-tooltip-beta >>> gux-tooltip-base-beta'
         );
         expect(baseTooltip.className.split(' ')).not.toContain('gux-show');
+        await a11yCheck(page);
```

Comment:
Added this to verify that there are no issues prior to the tooltip being activated.